db/migrate/20180618084054_init_zones_visibility.rb

```diff
@@ -0,0 +1,16 @@
+class InitZonesVisibility < ActiveRecord::Migration[5.0]
+  class Zone < ActiveRecord::Base
+  end
+
+  def up
+    say_with_time("Updating all zones to visible") do
+      Zone.update_all(:visible => true)
+    end
+  end
+
+  def down
+    say_with_time("Resetting zone visibility") do
```

Comment:
For things like this I think it should be merged with the `add_column` migration.  Then the `up` adds the column and updates the existing records and the `down` just drops the column.  Updating the column on every record before dropping the column seems pointless to me.

@carbonin @agrare Thoughts?

Reply:
@bdunne - there is a rule "Don't mix structural and data migration". So I followed it. 
If there are lots of reviewers they cannot want opposite things